### PR TITLE
Latest Camunda Version and Spring Boot, new handling of process variables

### PR DESCRIPTION
**Pull request by dbudke**

### CODE REVIEW

**Comment by dbudke** [comment]
please review build.gradle due to manual merge

**Comment by dbudke** [comment]
all tests ran successfully

File: src/main/java/ch/umb/solutions/camunda/helper/ProcessVariable.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package ch.umb.solutions.camunda.helper;
+
+import org.camunda.bpm.engine.delegate.VariableScope;
+
+public class ProcessVariable<T> {
```
Comment (feberle) [inline src/main/java/ch/umb/solutions/camunda/helper/ProcessVariable.java:5]: add private constructor to enforce usage of static create method

File: src/main/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/delegate/LoggerDelegate.java [hunk 2]
```diff
@@ -18,8 +18,8 @@
     final static Logger logger = LoggerFactory.getLogger(LoggerDelegate.class);
 
     @Override
-    public void execute(DelegateExecution execution)  {
-        String logMessage = (String) execution.getVariable(VARIABLE_LOGMESSAGE);
+    public void execute(DelegateExecution execution) {
```
Comment (feberle) [inline src/main/java/ch/umb/solutions/consulting/camundaspringbootquickstarter/delegate/LoggerDelegate.java:21]: add example UnitTest to show how to mock Variables

**Review by feberle — APPROVED** [review]

**Pull request merged**